dataikuapi/dss/notebookcontent.py

```diff
@@ -0,0 +1,46 @@
+from .discussion import DSSObjectDiscussions
+
+class DSSNotebookContent(object):
```

Comment:
Nitpick/Not sure: We could keep it in `notebook.py` like we do for other objects. Don't know what's best

Reply:
I will move everything to "jupyterNotebook.py", I don't want to merge legacy and non-legacy code.